File: app/src/main/java/com/neome/java/api/app/base/dto/DtoNeoScriptFormula.java
// neome.ai API - do not change
//

package com.neome.java.api.app.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdForm;
import com.neome.java.api.meta.base.Types.MetaIdFormula;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DtoNeoScriptFormula extends DtoNeoScript
{
  @Nullable
  public MetaIdForm formId;

  @Nullable
  public MetaIdFormula formulaId;
}

File: app/src/main/java/com/neome/api/app/base/dto/DtoNeoScriptFormula.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.app.base.dto

import com.neome.api.app.base.dto.DtoNeoScript
import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdFormula

interface DtoNeoScriptFormula : DtoNeoScript
{
  val formId: MetaIdForm?
  val formulaId: MetaIdFormula?
}